pyproject.toml

```
@@ -22,7 +22,8 @@ requires-python = ">=3.11"
 version = "0.0.0"
 
 [project.urls]
-repository = "https://github.com/quantco/dataframely"
+Repository = "https://github.com/quantco/dataframely"
+Documentation = "https://dataframely.readthedocs.io/"
```

The [documentation](https://packaging.python.org/en/latest/specifications/well-known-project-urls/#well-known-labels) lists the well-known labels as lowercase, I guess this would be more fitting:

```suggestion
repository = "https://github.com/quantco/dataframely"
documentation = "https://dataframely.readthedocs.io/"
```

---

On pypi, it's shown verbatim as written in that key. Your repo is the only one I've ever seen where those are written as lowercase. Looks out-of-place to me.

Not sure why they're lowercase in the docs there - super weird. Kinda seems like the pyproject docs suggest they're case-insensitive (and thus lowercase is the default), while PyPI (the website) interprets them as case-sensitive.

I stand by keeping them titlecase

---

Thanks :)